README.md

```diff
@@ -4,13 +4,26 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. The smaller order variables are ignored.
+  2. Coefficents are ignored.
+  3. Asymptotic analysis considers worst, average, and best times which can be misleading by overestimating or underestimating the time of the program.
+
 - Suppose finding a particular element in a binary search tree with 1,000
   elements takes 5 seconds. Given what you know about the asymptotic complexity
   of search in a binary search tree, how long would you guess finding the same
   element in a search tree with 10,000 elements takes? Explain your reasoning.
 
+  i would guess that it would take around 6.66 seconds, because we know searching 
+  through a binary tree takes log(n) time so we therefore know that 5 = C*log(1000).
+  so we just have to solve for c which gets us 0.501, then we use that in the equation
+  and solve it 0.501*log(10000) which gets us 6.66 seconds or so.
+
 - You measure the time with 10,000 elements and it takes 100 seconds! List 3
   reasons why this could be the case, given that reasoning with the asymptotic
   complexity suggests a different time.
 
+  1. When the program was initialy run it could have been run on a nasa super computer and gotten lucky when finding the number. Then it could have been run again on a computer from 1991 leading to much slower results.
+  2. The program does arbitrary things as it gets larger slowing down time.
+  3. The program for the larger set could have been given a linked list, and the program for the smaller set could have been given a binary tree. this could lead to the large descrepency.
```

Comment:
Don't see that -- a linked list would be linear, i.e. I would expect the time to be 10 times as much for 10 times the input. That's 50 seconds, and a long way off 100 seconds.

Reply:
it would take 10 times the amount of time if both times it was handed a linked list, but if the larger one is given a linked list while the smaller one is given a tree then the smaller one would take log(n) time, but the larger one would take n time.